Testnet/Ticketbooth/Ticketbooth/TicketContract_1_0_0.cs

```diff
@@ -0,0 +1,474 @@
+﻿using Stratis.SmartContracts;
+
+public class TicketContract_1_0_0 : SmartContract
+{
+    /// <summary>
+    /// Creates a new ticketing contract
+    /// </summary>
+    /// <param name="smartContractState"></param>
+    /// <param name="seatsBytes">The serialized array of seats</param>
+    /// <param name="venueName">The venue that hosts the contract</param>
+    public TicketContract_1_0_0(ISmartContractState smartContractState, byte[] seatsBytes, string venueName)
+        : base(smartContractState)
+    {
+        var seats = Serializer.ToArray<Seat>(seatsBytes);
+        var tickets = new Ticket[seats.Length];
+
+        for (int i = 0; i < seats.Length; i++)
+        {
+            tickets[i] = new Ticket { Seat = seats[i] };
+        }
+
+        Log(new Venue { Name = venueName });
+        Owner = Message.Sender;
+        Tickets = tickets;
+    }
+
+    /// <summary>
+    /// Stores ticket data for the contract
+    /// </summary>
+    public Ticket[] Tickets
+    {
+        get
+        {
+            return PersistentState.GetArray<Ticket>(nameof(Tickets));
+        }
+        private set
+        {
+            PersistentState.SetArray(nameof(Tickets), value);
+        }
+    }
+
+    private ulong EndOfSale
+    {
+        get
+        {
+            return PersistentState.GetUInt64(nameof(EndOfSale));
+        }
+        set
+        {
+            PersistentState.SetUInt64(nameof(EndOfSale), value);
+        }
+    }
+
+    private ulong ReleaseFee
+    {
+        get
+        {
+            return PersistentState.GetUInt64(nameof(ReleaseFee));
+        }
+        set
+        {
+            PersistentState.SetUInt64(nameof(ReleaseFee), value);
+            Log(new TicketReleaseFee { Amount = value });
+        }
+    }
+
+    private ulong NoRefundBlockCount
+    {
+        get
+        {
+            return PersistentState.GetUInt64(nameof(NoRefundBlockCount));
+        }
+        set
+        {
+            PersistentState.SetUInt64(nameof(NoRefundBlockCount), value);
+            Log(new NoRefundBlocks { Count = value });
+        }
+    }
+
+    private bool RequireIdentityVerification
+    {
+        get
+        {
+            return PersistentState.GetBool(nameof(RequireIdentityVerification));
+        }
+        set
+        {
+            PersistentState.SetBool(nameof(RequireIdentityVerification), value);
+            Log(new IdentityVerificationPolicy { RequireIdentityVerification = value });
+        }
+    }
+
+    private Address Owner
+    {
+        get
+        {
+            return PersistentState.GetAddress(nameof(Owner));
+        }
+        set
+        {
+            PersistentState.SetAddress(nameof(Owner), value);
+        }
+    }
```

Comment:
While code design is your choice, it might be beneficial to alter these for readability.

```dotnet
private Address Owner
{
    get => PersistentState.GetAddress(nameof(Owner));
    set => PersistentState.SetAddress(nameof(Owner), value);
}
```

Reply:
Agreed, I'll update this